docs/reference/support-policy.md

```diff
@@ -32,9 +32,6 @@ The current status of each Stencil version is:
 
 **Maintenance Period**: Only critical bug and security fixes. No major feature improvements.
 
-**Extended Support Period**: For teams and organizations that require additional long term support, Ionic has extended 
```

Comment:
FYI, I think we'll need to keep this around for past Enterprise customers. This snippet (and the above table) is the current source of truth for these things. Probably will just need a reword to refer to Enterprise in the past tense

Reply:
Updated this after having a conversation with Tori 👍